cfme/control/explorer/conditions.py

```diff
@@ -259,7 +261,7 @@ class ConditionCollection(BaseCollection):
     def create(self, condition_class, description, expression=None, scope=None, notes=None):
         condition = condition_class(self, description, expression=expression, scope=scope,
             notes=notes)
-        view = navigate_to(condition, "Add")
+        view = navigate_to(condition, "Add", wait_for_view=20)
```

Comment:
is the default of 10s not long enough?

Reply:
10s is probably fine, I can change it. Was doing some testing to see if 20 would help a test to pass, but the fix was due to something elsewhere and I never cleaned this up.